aiida_nanotech_empa/workflows/cp2k/bsse_workchain.py

```diff
@@ -0,0 +1,253 @@
+import os
+import pathlib
+import yaml
+import copy
+
+from aiida.engine import WorkChain, ToContext, ExitCode, calcfunction
+from aiida.orm import Int, Bool, Code, Dict, List, Str
+from aiida.orm import SinglefileData, StructureData
+from aiida.plugins import WorkflowFactory
+from aiida_nanotech_empa.workflows.cp2k.cp2k_utils import get_kinds_section, determine_kinds, dict_merge, get_nodes, get_cutoff
+from aiida_nanotech_empa.utils import analyze_structure
+
+from aiida_nanotech_empa.utils import common_utils
+
+Cp2kBaseWorkChain = WorkflowFactory('cp2k.base')
+
+
+@calcfunction
+def split_structure(charges, fragments, multiplicity):
```

Comment:
I think we really need to bring all these "split_structure" methods into one place. Probably "utils" is the best destination for it.